stdlib/set.ml

```diff
@@ -497,10 +497,10 @@ module Make(Ord: OrderedType) =
       let rec sub n l =
         match n, l with
         | 0, l -> Empty, l
-        | 1, x0 :: l -> Node (Empty, x0, Empty, 1), l
-        | 2, x0 :: x1 :: l -> Node (Node(Empty, x0, Empty, 1), x1, Empty, 2), l
+        | 1, x0 :: l -> Node {l=Empty; v=x0; r=Empty; h=1}, l
+        | 2, x0 :: x1 :: l -> Node{l=Node{l=Empty; v=x0; r=Empty; h=1}; v=x1; r=Empty; h=2}, l
```

Comment:
Is there a secret to be able to spot lines with more than 80 columns in github interface ?